### DISCUSSION TITLE
Projects view has incomprehensible label

### DISCUSSION THREAD

**Discussion by peterroth** [community/community#103036] [Projects and Issues] [answered]
Hi everyone,  
I just saw how Projects appear in the GitHub app (in my case on Android) and would like to know what is "view 4"? Other than that, how can I see tickets that are already started or finished, because I see only the Backlog label.![Screenshot_20240201_212044_com.github.android_edit_921468575978141.jpg](https://github.com/community/community/assets/35808401/858321d4-7145-4451-a6e6-98cfcc60774f)

**Comment by queenofcorgis** [comment]
Hi there, 

* Is View 4 just viewable on mobile or can you view across devices? 
* Do you have any filters in place that would only show items with the `Backlog` label?

**Reply by peterroth** [reply]
Hi @queenofcorgis,
Thank you for getting back to me.
- Where should I see the "View 4" in a browser? I didn't find it in Edge, checking the project on GitHub.
- The project has New, Backlog, Ready, In progress, In review, Done labels. So a few other.

**Reply by queenofcorgis** [reply, marked as answer]
<img width="1337" alt="Screenshot 2024-02-05 at 12 11 55 PM" src="https://github.com/community/community/assets/92824518/0b0eb2d8-de28-4f69-9bcb-d65ad1e1848f">

`View 4` would be in the tab view like `Board` and `Table`

For the labels do you have any views that would just show one of those labels?

**Reply by peterroth** [reply]
Yes, that's correct. I can see `View 4` as a view. I didn't notice it earlier.
Also, the main board is `Backlog`, here I can see the all the states of the tickets.
Mystery solved, thanks a lot!

Only one small question: in the `Backlog` view I see "Board layout is currently not supported.", will it be?

**Reply by queenofcorgis** [reply]
@peterroth I'm so glad I could help 🤠  thanks for troubleshooting with me. Would you mind sharing what type of board layout you have in `Backlog` so I could check in on this for you?

**Reply by peterroth** [reply]
We have Table, Board and Roadmap.  The Board is the default.

**Reply by queenofcorgis** [reply]
@peterroth would you mind sharing the org and/or repo name the project is in?

**Reply by queenofcorgis** [reply]
And where you are seeing the error?

**Reply by peterroth** [reply]
Yes, that might be easier. The project is: https://github.com/orgs/okd-project/projects/1/views/1

The error appears only in the official GitHub app (in my case on Android).

**Reply by queenofcorgis** [reply]
@peterroth We currently only support `Table` layout on mobile, not `Board` or `Roadmap`
